src/public/assets/lang/ru.yaml

```diff
@@ -1,145 +1,146 @@
 ---
 # Global
 # footer.tsx
-overview:
-home:
-features:
-downloads:
-external links:
+overview: Обзор
+home: дом
+features: Особенности
+downloads: Загрузки
+external links: внешние ссылки
 twitter:
 github:
 youtube:
 discord:
 reddit:
-legal:
-license:
-eula:
-history of incidents:
-privacy and security:
-inspired by:
-avdanos contributors:
+legal: юридический
+license: лицензия
+eula: Эла
+history of incidents: История инцидентов
+privacy and security: Конфиденциальность и безопасность
+inspired by: вдохновлен
+avdanos contributors: Авданос участники
```

Comment:
word contributor should be used instead uchastniki (it is more common to use contributors)